### ISSUE TITLE
Get Local position of Gesture Detector

### ISSUE THREAD

**Issue by rickyazhari** [flutter/flutter#20986]
i have problems to get real local position on gesture detector widget using pan update

here is my sample code

```dart
new Center(    
      child new Container(
           height: 150.0,
           width: 150.0,
           decoration: new BoxDecoration(
             borderRadius: new BorderRadius.circular(5.0),
             color: Colors.white,
          ),
          child: new GestureDetector(
          onPanUpdate: (details) => onPanUpdate(context, details),
          child: new CustomPaint(
             painter: new MyPainter(
                 points: points,
                 limitBottom:height - 125.0,
                 limitTop: 10.0,
                 limitLeft: 5.0,
                 limitRight: width - 55.0
             ),
           ),
        ),
      ),  
    )
```
when i print the the Offset of global position && local position, i've got same value for both of them. in Result it was painted outside of my Container widget. is there anything i missed to get local position.

my flutter doctor

```
ITID000419-MAC:tanamduit-app-ios lumos$ /Users/lumos/personal/sdk/flutter/bin/flutter doctor
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel beta, v0.5.1, on Mac OS X 10.13.4 17E202, locale en-ID)
[✓] Android toolchain - develop for Android devices (Android SDK 27.0.3)
[!] iOS toolchain - develop for iOS devices (Xcode 9.4.1)
    ! CocoaPods out of date (1.5.0 is recommended).
        CocoaPods is used to retrieve the iOS platform side's plugin code that responds to your plugin usage on the Dart side.
        Without resolving iOS dependencies with CocoaPods, plugins will not work on iOS.
        For more info, see https://flutter.io/platform-plugins
      To upgrade:
        brew upgrade cocoapods
        pod setup
[✓] Android Studio (version 3.0)
[!] VS Code (version 1.26.1)
[✓] Connected devices (2 available)

! Doctor found issues in 2 categories.
ITID000419-MAC:tanamduit-app-ios lumos$
```

**Comment by rickyazhari** [comment]
@zoechi  thanks you are really pro active
- link one it's me
- link two  there is no specific size container.  so the size will be same as screen size.

i still have no idea to figure it how

**Comment by the4thfloor** [comment]
I got it by putting a `LayoutBilder` between `Container` and `GestureDetector` and using its `context` for `RenderBox box = context.findRenderObject()`.

**Comment by johnpryan** [comment]
I think this is fixed in #32192

**Comment by zoeyfan** [comment]
Closing this issue per the previous comment.
